services/director/requirements/_base.txt

```diff
@@ -4,22 +4,22 @@
 #
 #    pip-compile --output-file=_base.txt _base.in
 #
+aiodocker==0.14.0
 aiohttp==3.3.2
 git+https://github.com/ITISFoundation/aiohttp_apiset.git@fixes_4_osparc#egg=aiohttp_apiset
+async-generator==1.10     # via asyncio-extras
 async-timeout==3.0.1      # via aiohttp
+asyncio-extras==1.3.2
```

Comment:
we should perhaps update to python 3.7 sometime soon :-)
Async context managers, e.g. are already there https://docs.python.org/3/library/contextlib.html

Reply:
agree